homeassistant/components/epson/config_flow.py

```diff
@@ -25,18 +25,31 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_import(self, import_config):
         """Import a config entry from configuration.yaml."""
-        return await self.async_step_user(import_config)
+        for entry in self._async_current_entries(include_ignore=True):
+            if import_config[CONF_HOST] == entry.data[CONF_HOST]:
+                return self.async_abort(reason="already_configured")
+        await self.async_set_unique_id()
+        self._abort_if_unique_id_configured()
+        return self.async_create_entry(
```

Comment:
We should make the same connection checks here before creating a new entry as when starting a manual flow in the user step. A common pattern is to just pass the imported config on to the user step.